src/features/embalm/stepContent/components/ArchaeologistHeader.tsx

```diff
@@ -0,0 +1,68 @@
+import { Text, Box, Flex, useColorModeValue, HStack, Checkbox, Icon } from '@chakra-ui/react';
+import { InfoOutlineIcon } from '@chakra-ui/icons';
+import { sumDiggingFees } from 'lib/utils/helpers';
+import { useDispatch, useSelector } from 'store/index';
+import { setShowSelectedArchaeologists } from 'store/embalm/actions';
+
+export function ArchaeologistHeader() {
+  const dispatch = useDispatch();
+  const { selectedArchaeologists, ShowSelectedArchaeologists } = useSelector(x => x.embalmState);
+
+  return (
+    <Box mt={6}>
+      <Flex
+        py={'13.5'}
+        border={1}
+        borderColor={useColorModeValue('gray.200', 'gray.900')}
```

Comment:
I doubt we'll ever have a light theme for sarcophagus, but it may be something worth talking about?

Reply:
Yes ok, we can talk about it